examples/config/perfcomp/json/deployer.json

```
@@ -0,0 +1,86 @@
+  "BrowserAgentInstanceType": "c7i.xlarge",
+  "AppInstanceCount": 2,
+  "AppInstanceType": "c7i.2xlarge",
+  "ClusterName": "TBD",
+  "LoadTestDownloadURL": "https://github.com/mattermost/mattermost-load-test-ng/releases/download/vTBD/mattermost-load-test-ng-vTBD-linux-amd64.tar.gz",
+  "MattermostDownloadURL": "https://latest.mattermost.com/mattermost-enterprise-linux",
+  "MattermostLicenseFile": "",
+  "MetricsInstanceType": "t3.xlarge",
+  "ProxyInstanceCount": 1,
+  "ProxyInstanceType": "c7i.xlarge",
+  "SSHPublicKey": "~/.ssh/id_ed25519.pub",
+  "TerraformStateDir": "./ltstatus",
```

Biome (2.4.6)

[error] The key TerraformStateDir was already declared.

(lint/suspicious/noDuplicateObjectKeys)